aioredis/connection.py

```diff
@@ -679,7 +679,7 @@ async def _connect(self):
         """Create a TCP socket connection"""
         async with async_timeout.timeout(self.socket_connect_timeout):
             reader, writer = await asyncio.open_connection(
-                host=self.host, port=self.port, ssl=self.ssl_context, loop=self._loop
+                host=self.host, port=self.port, ssl=self.ssl_context
```

Comment:
Versions before Python 3.10 expect the loop argument to work though. Find a better way to input arguments without causing any warnings to appear but to still have compatibility with Python 3.9 and below

Reply:
I do not think python 3.9 or 3.8 iirc it was only for older python versions

Reply:
@Andrew-Chen-Wang yeah, it was only for older python: https://stackoverflow.com/questions/60312374/what-are-all-these-deprecated-loop-parameters-in-asyncio